recipe/meta.yaml

```diff
@@ -38,129 +44,147 @@ build:
-    - cairo                    # [not win]
-    - libssh2                  # [not win]
-    - openssl                  # [not win]
-    - krb5                     # [not win]
-    - {{native}}bzip2 1.0.*    # [not osx]
-    - {{native}}libjpeg-turbo  # [win]
-    - {{native}}libiconv       # [win]
-    - {{native}}gmp 6.1.*      # [win]
-    - {{native}}fftw           # [win]
-    - {{native}}xz             # [win]
-    - {{native}}mpfr 3.1.*     # [win]
-    - {{native}}libsndfile     # [win]
-    - {{native}}bwidget        # [win]
-    - {{native}}tktable        # [win]
-    - {{native}}libpng         # [win]
-    - pango                    # [not win]
-    - icu                      # [not win]
-    - pcre                     # [not win]
-    - {{native}}pcre           # [win]
-    - {{native}}tk             # [win]
-    - tk                       # [not win]
-    - zlib                     # [not win]
-    - {{native}}zlib           # [win]
-    - {{native}}gsl            # [win]
-    - gsl                      # [not win]
-    - libtiff                  # [not win]
-    - {{native}}libtiff        # [win]
-    - {{native}}libxml2 2.9.*  # [win]
-    - libxml2                  # [not win]
-    - xorg-libx11              # [linux]
-    - xorg-libxt               # [linux]
-    - xorg-libxrender          # [linux]
-    - xorg-libxext             # [linux]
-    - xorg-libxdmcp            # [linux]
+    - _r-mutex 1.* anacondar_1
+    # Needed by R CMD javareconf
+    - make                               # [not win]
+    - {{ compiler('fortran') }}          # [not win]
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - llvm-openmp >=4.0.1                # [osx]
+    - {{native}}gcc-libs                 # [win]
+    - curl                               # [not win]
+    - {{native}}bzip2 1.0.*              # [win]
+    - {{native}}libjpeg-turbo            # [win]
+    - {{native}}libiconv                 # [win]
+    - {{native}}libuuid                  # [linux]
+    - {{native}}gmp                      # [win]
+    - {{native}}fftw                     # [win]
+    - {{native}}xz                       # [win]
+    - {{native}}mpfr                     # [win]
+    - {{native}}libsndfile               # [win]
+    - {{native}}bwidget
+    - {{native}}tktable
+    - {{native}}libpng                   # [win]
+    - {{native}}pcre                     # [win]
+    - {{native}}tk                       # [win]
+    - {{native}}zlib 1.2.8               # [win]
+    - {{native}}gsl                      # [win]
+    - {{native}}libtiff >=4.0.3,<4.0.8   # [win]
+    - {{native}}libxml2 2.9.*            # [win]
+    - {{native}}openblas                 # [win]
+    # Once run_exports are in place these will not be necessary:
+    - {{ pin_compatible('pango') }}      # [not win]
+    - {{ pin_compatible('bzip2') }}      # [not win]
+    - {{ pin_compatible('icu') }}        $ [not win]
```

Comment:
@mingwandroid this is the typo. Also there is a meta.yaml.orig included in this PR.

Reply:
Urgh! How did I not spot that (and why is it not happening locally .. questions for myself)? OK. thanks.